doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_loggingr.rst

```diff
@@ -71,6 +73,7 @@ This method bypasses the need for string formatting functions and peripheral dri
 Configuration
 -------------
 
+.. to review
 To show the logs for a given domain in the UART output, you must enable the STM for that domain.
```

Comment:
Configuration of STM logging will change but it is not yet concluded how. So far we relied on rebuilding SDFW with STM support but it will change to UICR configuration (so that same SDFW can be used). This will require device tree configuration to enable STM logging. I've started with a snippet but PR is still in review so we might end up with something else.